### PR TITLE
Add FFMPEG Vulkan decoder for H264 and H265

### PR DESCRIPTION
**Pull request by ceyusa**
Right now it's possible to test FFmpeg Vulkan decoding by using https://github.com/cyanreg/FFmpeg.git branch vulkan

It's required to process frames in vulkan output and then download them in it's native format, to later convert them to the required format for validation.

It requires the latest Vulkan beta driver from NVIDIA or radv/anv drivers from MESA (current top of tree).

### CODE REVIEW

File: fluster/decoders/ffmpeg.py [hunk 1]
```diff
@@ -27,8 +27,19 @@
...
+def output_format_to_ffformat(output_format: OutputFormat) -> str:
+    """Return GStreamer pixel format"""
+    mapping = {
+        OutputFormat.YUV420P: "nv12",
+        OutputFormat.YUV422P: "nv12", # vulkan
+        OutputFormat.YUV420P10LE: "p010",
+        OutputFormat.YUV422P10LE: "p012",
+    }
+    if output_format not in mapping:
+        raise Exception(f"No matching output format found in FFMpeg for {output_format}")
```
Comment (rgonzalezfluendo) [inline fluster/decoders/ffmpeg.py:41]:
```suggestion
        raise Exception(f"No matching output format found in FFmpeg for {output_format}")
```

I see `FFMpeg` on many sites, but I don't know what the origin is. I think correct is `FFmpeg`.

**Comment by rgonzalezfluendo** [comment]
Thanks for the contribution. Looking forward for `GstVulkanDeviceDecoder`.

File: fluster/decoders/ffmpeg.py (hunk 1 above)
Comment (ceyusa) [reply fluster/decoders/ffmpeg.py]: thanks :)

**Comment by ceyusa** [comment]
now I ran `make check` locally :)

**Review by rgonzalezfluendo — APPROVED** [review]

**Comment by rgonzalezfluendo** [comment]
Thanks

**Pull request merged**